sql/_05_plcsql/_01_testspec/_03_statement/_07_exit/_01_with_label/cases/03_07_01-01_normal_basic.sql

```diff
@@ -0,0 +1,29 @@
+--+ server-message on
+
+-- normal: basic usage of exit statements with labels
+
+
+create or replace procedure t(i int) as
+begin
+    <<outer>>
```

Comment:
“Outer” and “inner” are reserved words. Can they be used as label names?

Reply:
Some reserved words of CUBRID are not reserved of PL/CSQL, while they are reserved words if they appear in a Static/Dynamic SQL.
PL/CSQL reserved words are listed in http://192.168.2.208/ko/pl.html#id3, although they need some updates.